core/src/main/resources/AppConstants.properties

```diff
@@ -39,6 +39,8 @@ configurations.validate=true
 
 adapters.autoStart=true
 
+# Change to nl.nn.adapterframework.configuration.AdapterService in order to disable JMX
+configurations.adapterService=nl.nn.adapterframework.configuration.JmxRegisteringAdapterService
```

Comment:
Ik vind het eigenlijk geen goed idee dat we nu ons vastbinden op 'AdapterService'.
Kunnen we dit niet doen op basis van jmx.enabled=true of false?